folsom-opencensus/src/main/java/com/spotify/folsom/opencensus/OpenCensus.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.spotify.folsom.opencensus;
+
+import com.spotify.folsom.Tracer;
+
+/** Starting point for the OpenCensus tracing implementation for Folsom */
+public class OpenCensus {
+
+  private static final boolean DEFAULT_INCLUDE_KEYS = true;
+  private static final boolean DEFAULT_INCLUDE_VALUES = false;
```

Comment:
I'm a bit concerned about someone enabling the value attribute. It could be a really big payload. I think since folks have the ability to see the key if they really care about the value they can fetch it themselves.

I would be in favor of removing the value attribute entirely.

Reply:
I think it's fine as long as this is default to off, but I'll add a note about this in the documentation.